lib/SPIRV/SPIRVReader.cpp

```diff
@@ -987,6 +995,41 @@ void SPIRVToLLVM::setLLVMLoopMetadata(const LoopInstType *LM,
   }
   if (LC & LoopControlNoFusionINTELMask)
     Metadata.push_back(getMetadataFromName("llvm.loop.fusion.disable"));
+
+  if (LC & spv::internal::LoopControlLoopCountINTELMask) {
+    uint64_t LoopCountMin = 0;
+    LoopCountMin = static_cast<uint64_t>(LoopControlParameters[NumParam++]);
```

Comment:
`uint64_t LoopCountMin = LoopCountMin = static_cast<uint64_t>(LoopControlParameters[NumParam++]);`

same for Max and Avg